resources/views/components/game/compare-progress.blade.php

```diff
@@ -24,17 +34,21 @@
         'last_played_at',
     ];
 
-    $followedUserCompletion = PlayerGame::where('game_id', $game->ID)
-        ->whereIn('user_id', $followedUserIds)
+    $followedUserCompletion = PlayerGame::where('game_id', $game->id)
+        ->whereIn('user_id', $limitedFollowedUsers)
         ->where(function ($query) {
             $query->where('achievements_unlocked', '>', 0)
                 ->orWhere('achievements_unlocked_hardcore', '>', 0);
         })
         ->select($fields)
         ->orderBy('achievements_unlocked_hardcore', 'DESC')
         ->orderBy('achievements_unlocked', 'DESC')
+        ->limit(50)
```

Comment:
With 1000 potential matches on a popular set, it's highly probable that more than 50 of them may have mastered it. You should have a third `orderBy` clause - maybe `last_unlock_at` or `last_played_at`?

Reply:
Added a third order by in latest by `last_unlock_at` (desc).

Reply:
Actually just flipped this from desc to asc. I figure asc probably makes more sense for this ordering.

Reply:
I think `desc` makes more sense. I think I'd rather see recently active people over people who played the game years ago. It's kind of like the "Latest Masters" vs "Top 10" list we used to have.

But unless you're looking at a game that more than 50 of your followed users have mastered, it probably doesn't matter.

Reply:
Swapped back to descending in latest.